app-backend/mongo/data/mongo/user.js

```diff
@@ -0,0 +1,93 @@
+const validator = require('validator');
+const UserModel = require('../../model/User.js');
+const moment = require('moment');
+const fs = require('fs');
+
+const createErrorResponse = (statusCode, message) => ({
+    statusCode: statusCode || 501,
+    headers: { 'Content-Type': 'text-plain' },
+    body: message || 'Incorrect id'
+});
+
+export function getUsers() {
+
+    return new Promise((resolve, reject) => {
+        UserModel
+            .find()
+            .then((users) => {
+                let userMap = [];
+                users.forEach((user, index) => {
+                    userMap[index] = user;
+                });
+                let res = {};
+                res.users = userMap;
+                resolve(res.users);
+            })
+            .catch((err) => {
+                reject(err);
+            });
+        }
+    );
+}
+
+export function getUserById(id) {
+    return new Promise((resolve, reject) =>
+        {
+            UserModel
+                .findOne({_id: id})
+                .then((user) => {
+                    resolve(user);
+                })
+                .catch((err) => {
+                    reject(err);
+                })
+                .finally(() => {
+                    // db.close();
```

Comment:
Is this dead code? If so please remove it.

Reply:
will remove.